### PR TITLE
Bump k8s 1.27.2

### PR DESCRIPTION
**Pull request by dinhxuanvu**
Bump k8s 1.27.2

### CODE REVIEW

**Comment by dinhxuanvu** [comment]
Proof PR: https://github.com/openshift/openshift-apiserver/pull/371

**Comment by dinhxuanvu** [comment]
@soltysh Please take a look. The cherry-picking of carry patches was smooth with one exception which I will tag you on the code for verification. Thanks.

File: pkg/storage/value/encrypt/aes/aes_test.go [hunk 1]
```diff
@@ -669,10 +671,10 @@ func TestRoundTrip(t *testing.T) {
 		t       value.Transformer
 	}{
 		{name: "GCM 16 byte key", t: newGCMTransformer(t, aes16block)},
-		{name: "GCM 24 byte key", t: newGCMTransformer(t, aes24block)},
+		// FIPS disabled {name: "GCM 24 byte key", t: NewGCMTransformer(aes24block)},
 		{name: "GCM 32 byte key", t: newGCMTransformer(t, aes32block)},
 		{name: "GCM 16 byte unsafe key", t: newGCMTransformerWithUniqueKeyUnsafeTest(t, aes16block)},
-		{name: "GCM 24 byte unsafe key", t: newGCMTransformerWithUniqueKeyUnsafeTest(t, aes24block)},
+		// FIPS disabled {name: "GCM 24 byte unsafe key", t: newGCMTransformerWithUniqueKeyUnsafeTest(t, aes24block)},
```
Comment (dinhxuanvu) [inline pkg/storage/value/encrypt/aes/aes_test.go:677]: @soltysh They added a bunch of GCM unsafe key options. I disabled the GCM 24 byte unsafe key as well.

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**